### ISSUE TITLE
Function names instead of node indices when printing

### ISSUE THREAD

**Issue by anomam** [cedricleroy/pyungo#1]
It would be nice to be able to display the function names instead of the node indices when printing the graph dependencies.

**Comment by cedricleroy** [comment]
I agree. The way to do that today is:
```python
for node_ids in topological_sort(graph._dependencies()):
    nodes = [graph._get_node(node_id) for node_id in node_ids]
    print(nodes)
```
It should probably be a core feature.

**Comment by cedricleroy** [comment]
Fixed by #7
